src/common/ErrorBoundary.test.tsx

```diff
@@ -10,8 +10,11 @@ describe('Children throw an error', () => {
   let element
   let error
   let errorReportingService
+  let spy
 
   beforeEach(async () => {
+    spy = jest.spyOn(console, 'error')
+    spy.mockImplementation()
```

Comment:
Now that `error` is spied, you could also verify it was called or not called correctly.